logging/cluster-logging-external.adoc

```diff
@@ -1,6 +1,6 @@
 :context: cluster-logging-external
 [id="cluster-logging-external"]
-= Forwarding logs to third party systems
+= Forwarding logs to third-party systems
 include::modules/common-attributes.adoc[]
 
 toc::[]
```

Comment:
Can't comment on line 10:
You can also enable TLS support to...

Should TLS be expanded here? Or maybe it's obvious to everyone who would be interested in this set of docs?

Reply:
Updated. Thanks.